db/new_seeds/base/add_mentor_scenarios.rb

```diff
@@ -80,10 +80,11 @@
   ),
 ].each do |mentor_params|
   NewSeeds::Scenarios::Participants::Mentors::MentorWithNoEcts
-    .new(full_name: mentor_params.full_name)
+    .new(school_cohort: mentor_params.school_cohort, full_name: mentor_params.full_name)
     .build(
-      school_cohort: mentor_params.school_cohort,
       schedule: mentor_params.schedule,
     )
+    .chain_add_validation_data
```

Comment:
How do you feel about renaming these to `.with_validation_data` and `.with_eligibility`?  Think it would flow nicer when reading the code

Reply:
I think it is a good idea